exampleCourse/questions/examplesRAddTwoNumbers/server.py

```diff
@@ -0,0 +1,26 @@
+# Access R using rpy2
+import rpy2.robjects as robjects
+
+def generate(data):
+
+    # Sample two random integers between 5 and 10 (inclusive)
+    values = robjects.r(
+      '''
+       x = runif(2, 5, 10)
+       x = floor(x)
+       x
+      '''
+    )
+    # Cast to R
+    a = values[0] + 0
+    b = values[1] + 0
```

Comment:
What’s the `+ 0` doing here? In my tests it seems unnecessary, because `values[0]` already has type float.